src/library/dlgtagfetcher.cpp

```diff
@@ -222,6 +234,29 @@ void DlgTagFetcher::apply() {
             QDateTime::currentDateTimeUtc());
 }
 
+void DlgTagFetcher::retry() {
+    results->clear();
+    disconnect(m_track.get(),
```

Comment:
Can we remove the connect/reconnect cycle?

Reply:
Yes, we can. That was added in the first commits when the retry button was always available even for success results. 

For now this is no needed IMHO.